### ISSUE TITLE
Bump quarkus to 3.0 and  hypersistence-utils to 3.2.0

### ISSUE THREAD

**Issue by pipinet** [quarkiverse/quarkus-hibernate-types#123]
Quarkus 3.0.0.Alpha5 is out, so hibernate is bump to 6.2。
hypersistence-utils seems changed name and package from hibernate types.

**Comment by gastaldi** [comment]
Support has been added in https://github.com/vladmihalcea/hypersistence-utils/tree/master/hypersistence-utils-hibernate-62

Looking forward to this so I can migrate registry.quarkus.io to Quarkus 3 😉

**Comment by gastaldi** [comment]
Also, the project name was changed from Hibernate Types to Hypersistence Utils. What is the strategy here: rename the repository or create a new repository and archive this one?

**Comment by pipinet** [comment]
I feel like upgrading Hibernate 6.2 will no longer require quarkus-hibernate-types and hypersistence-utils

**Comment by gastaldi** [comment]
Interesting, I am looking for the JSON support and also the following naming strategy:

```properties
quarkus.hibernate-orm.physical-naming-strategy=com.vladmihalcea.hibernate.naming.CamelCaseToSnakeCaseNamingStrategy
```

**Comment by HerrDerb** [comment]
Even if you suggest that the extension is now replaceable, it still would be convenient for us if you would upgrade your extension, so we can do the replacement step by step

**Comment by roman-svystun** [comment]
With Quarkus 3 released yesterday, it would be amazing to have this wonderful extension release a new version.

**Comment by wglanzer** [comment]
It seems that the extension is necessary, so I looked into my PR #128 again and I think I found a way to provide everything needed.
I'll release a `v2.0.0` in a few moments containing all those fixes, if the CI build (https://github.com/quarkiverse/quarkus-hibernate-types/actions/runs/4822413502/jobs/8589617031?pr=128) is successful.
Are you guys able to try it in a real world scenario and make sure that it works as intended?
